modules/virt-vm-creating-nic-web.adoc

```diff
@@ -2,8 +2,8 @@
 //
 // * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
 
-[id="virt-vm-create-nic-web_{context}"]
-= Creating a NIC for a virtual machine
+[id="virt-vm-creating-nic-web_{context}"]
+= Creating a NIC for a virtual machine in the web console
```

Comment:
is this followed with screen images that the user (and myself) can follow alongside the guide?

Reply:
We don't have any screen images due to the maintenance cost to keep up with UI every release.
We do have a breakdown of the fields in that window directly after this procedure:
https://deploy-preview-30387--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html#virt-networking-wizard-fields-web_virt-attaching-multiple-networks